app/assets/stylesheets/pageflow/ui/tooltip.scss

```diff
@@ -0,0 +1,57 @@
+.tooltip {
+  $background-color: rgba(0, 0, 0, 0.7);
+
+  position: absolute;
+  z-index: 10001;
+  top: 40px;
+  left: 400px;
+  max-width: 70%;
+  padding: 10px 15px;
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.